api/dsym.md

```diff
@@ -0,0 +1,23 @@
+## Upload your .dSYM files
+
```

Comment:
Somewhere in here (either as the description for the call, which would go here, or in the Example section), can we have some text explaining that the way this is usually used is via the upload_dsym script, with a link to the instructions for how to set that up?

I do see the link in the params table, but I think it's important enough to call it out in the main section.